gulp/tasks/webpack.js

```diff
@@ -0,0 +1,45 @@
+const gulp = require('gulp');
+const config = require('../config');
+const pi = require('gulp-load-plugins')();
+const path = require('path');
+const webpackStream = require('webpack-stream');
+const webpack2 = require('webpack');
+
+function webpack() {
+    return gulp.src([
+        `${config.dest}/app/main.js`
+    ])
+    .pipe(webpackStream({
+        // watch: true, // This causes gulp to freeze and not serve
+        output: {
+            path: path.resolve(config.dest),
+            filename: "bundle.js",
+            publicPath: "/", // for where to request chunks when the SinglePageApp changes the URL
+            chunkFilename: "chunk-[chunkhash].js"
+        },
+        externals: {
+            settings: 'SETTINGS'
+        },
+        module: {
+            rules: [
+                {
+                    test: /\.css$/,
+                    loader: 'ignore-loader'
+                }
+            ]
+        },
+        plugins: [
+            new webpack2.optimize.UglifyJsPlugin(),
+            new webpack2.optimize.MinChunkSizePlugin({minChunkSize: 16000})
```

Comment:
How was this size determined? Consider including a comment in the code so future-selves will know